components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -500,6 +480,25 @@ const InsightPage = ({ edit, insight, pageRepos }: InsightPageProps) => {
             );
           })}
         </RepositoriesCart>
+        <Title className="!text-1xl !leading-none mb-4 mt-8" level={4}>
+          Page Visibility
+        </Title>
+
+        <div className="flex justify-between">
+          <div className="flex items-center">
+            <UserGroupIcon className="w-[24px] h-[24px] text-light-slate-9" />
+            <Text className="pl-2">Make this page publicly visible</Text>
+          </div>
+
+          <div className="flex mx-4 !border-red-900 items-center">
```

Comment:
we should set the flex direction to `col` on mobile. the line brake in the word isn’t looking 👍

Reply:
🤔  Since we have the `Make this page publicly visible` word here, taking off the `Make Public` word might would not hurt in this case(Mobile only).

Reply:
Sure will  do that